controllers/collect.js

```diff
@@ -1,2 +1,17 @@
 // Listening for requests from webhooks
 // Sends the requests to our PG and Mongo databases
+const express = require('express');
+const router = express.Router();
+const bodyParser = require('body-parser');
```

Comment:
Can we just require this in app.js so that we don't have to repeat it in every route

Reply:
I thought that too, but if you remember in full stack open, we end up actually creating mini-routers for each entity. More [here](https://expressjs.com/en/guide/routing.html#:~:text=Use%20the%20express.Router%20class%20to%20create%20modular%2C%20mountable%20route%20handlers.%20A%20Router%20instance%20is%20a%20complete%20middleware%20and%20routing%20system%3B%20for%20this%20reason%2C%20it%20is%20often%20referred%20to%20as%20a%20%E2%80%9Cmini%2Dapp%E2%80%9D.).

Reply:
For `bodyParser`, we need to parse the body as "raw" for the collect API, but json for the other routes, so we can't have a catch-all in `app.js`.